relay-server/src/actors/project.rs

```diff
@@ -630,47 +635,49 @@ impl Project {
         Response::future(future)
     }
 
-    fn fetch_state(&mut self, no_cache: bool, context: &mut Context<Self>) {
+    pub fn update_state(
+        &mut self,
+        state_result: Result<ProjectStateResponse, ()>,
```

Comment:
_nit_: Let's make this an `Option` since the error does not have any data. This helps self-documenting the interface: You update the state, either by setting it or by explicitly removing it.

Reply:
ok